pkg/cmd/controller/pipelinerunner_integration_test.go

```diff
@@ -102,7 +103,7 @@ var _ = Describe("Pipeline Runner Integration", func() {
 			pipelineRunner := PipelineRunnerOptions{
 				CommonOptions:        &commonOpts,
 				Path:                 "/",
-				BindAddress:          "0.0.0.0",
+				BindAddress:          "127.0.0.1",
```

Comment:
@abayer - I am changing this to only bind to the loopback interface in the hope that this fixes the flakiness of the test. I tested this in a separate pull request and triggered the integration test multiple times and they passed.

If we continue seeing that this test is flaky we can take the next step and remove it.

cc @pmuir ^^

Reply:
+1

Reply:
Ok,  I removed it after all. The funny thing is that the unit test does the same test, but there it works. The difference is that the unit tests make no calls within the cluster.